src/System.IO/tests/Stream/TimeoutTests.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.IO;
+using Xunit;
+
+namespace StreamTests
+{
+    public class MyStream : Stream
```

Comment:
Is there a better name we could use for this, something that describes its purpose?